beheer.md

```diff
@@ -23,3 +24,65 @@ Bij de overdracht van een standaard van ontwikkeling naar beheer moeten ook de n
 Aangezien bij overdracht van ontwikkeling naar beheer de GitHub repository as-is wordt overgenomen kan de backlog zelf blijven waar deze is.
 
 Overdracht van de backlog en GitHub repository is een officieel moment en moet plaatsvinden voordat er sprake kan zijn van in beheername van een Open API
+
+## Werken met Docker
+
+Zie https://docs.docker.com/get-started/ en specifiek https://docs.docker.com/get-started/part2/ en https://docs.docker.com/get-started/part3/ voor een diepere uitleg over Docker.
+
+Voor het vervaardigen van Docker images heb je het volgende nodig:
+
+* een goed werkende Docker installatie;
+* een folder met daarin:
+  - een Dockerfile (met de naam 'Dockerfile');
+  - een requirements file (met de naam requirements.txt);
```

Comment:
```suggestion
```

Als het goed is staat Docker los van de "inner workings" van het component. Je hebt in principe **alleen** Docker nodig om het component te starten.

requirements.txt file is niet nodig voor Docker images maken, dit is een Python dingetje om libraries te installeren die nodig zijn voor de applicaties. Dit gebeurt allemaal **in** Docker dus niet op je eigen computer.